src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/UnsafeNativeMethods.cs

```diff
@@ -459,7 +459,6 @@ internal enum EvtSeekFlags
         internal static extern bool EvtCancel(EventLogHandle handle);
 
         [DllImport(WEVTAPI)]
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.